crates/edr_napi/test/provider.ts

```diff
@@ -415,3 +515,48 @@ function assertEqualMemory(stepMemory: Buffer | undefined, expected: Buffer) {
 
   assert.isTrue(stepMemory.equals(expected));
 }
+
+function assertJsonRpcFormatNormalised(trace: RpcDebugTraceResult) {
```

Comment:
I think these assertions can happen without knowing the type `RpcDebugTraceResult`, but am not 100%. If so, we can just remove the type from EDR's N-API, reducing our API surface.